pyteal/ast/itxn.py

```diff
@@ -111,10 +129,10 @@ def Submit(cls) -> Expr:
 
         Requires TEAL version 5 or higher. This operation is only permitted in application mode.
         """
-        return InnerTxnActionExpr(False)
+        return InnerTxnActionExpr(InnerTxnAction.Submit)
 
     @classmethod
-    def SetField(cls, field: TxnField, value: Expr) -> Expr:
+    def SetField(cls, field: TxnField, value: Union[Expr, List[Expr]]) -> Expr:
```

Comment:
It would probably be a good idea to have the docstrings for `SetField` and `SetFields` explain that for non-array fields, setting it again will overwrite the original value, but for array fields setting it again will append more values